pkg/labeler/config.go

```diff
@@ -27,8 +27,8 @@ type Config struct {
 	// The label with the highest score is applied to the issue.
 	Labels map[string]Label `yaml:"labels,omitempty"`
 
-	// RequiredLabels is a list of labels that must be present on the issue for any other labels to be applied.
-	RequiredLabels []string `yaml:"required_labels,omitempty"`
+	// RequireLabel is a list of labels that must be present on the issue for any other labels to be applied.
```

Comment:
Do we require all the labels in the list to be in the issue in order to process or is only one of them necessary? The comment should clarify that.

Reply:
good point, the `regex-labeler` requires them all to be present while the `ic-assigner` only requires one to be present.
I think we should make this consistent too, the `regex-labeler` should only require one of them to be present if any are configured.

Reply:
Update the PR.